docs/assignment1.md

```diff
@@ -1,62 +1,154 @@
-
-| ID  | Short name  | Description  |
-|---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+These functional features are imperative in order to make the game work the way is necessary. Many of these features are inherently linked with each other, therefore if one feature is not met in the final product, then the entire game falls apart. These features are inspired by what we know of adventure games, as well as certain features that are present in text-based games only, such as written commands and a turn-based time mechanic.
+
+<table>
+<tr>
+<th>ID</th>
+<th>Short Name</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>F1</td>
+<td>Commands</td>
+<td>Game shall have a <b>text-based option system</b>, where the player can type different commands in a command prompt which will result in different
+actions(F3).</td>
+</tr>
+<tr>
+<td>F2</td>
+<td>Interface</td>
+<td>The game shall be played inside a command prompt. Everything is text. Some scenarios should have ASCII art.</td>
+</tr>
+<tr>
+<td>F3</td>
+<td>Actions</td>
+<td>Typing different commands shal result in different actions in the game world, examples of such are:
+<ul>
+<li>Movement</li>
+<li>Attacking</li>
+<li>Conversation</li>
+<li>Interacting with items (doors, keys, picking up items)</li>
+<li>Dying (restarts the game)</li>
+<li>Healing</li>
+<li>Examining items and surrounding</li>
+<li>Sleeping</li>
+</ul>
+</td>
+</tr>
+<tr>
+<td>F4</td>
+<td>Items</td>
+<td>There shall be a multitude of different items which can be added to the inventory of the player (F8):
+<ul>
+<li>Weapons</li>
+<li>Keys</li>
+<li>Healing items</li>
+<li>Ammunition</li>
+</ul>
+</td>
+</tr>
+<tr>
+<td>F5</td>
+<td>Characters</td>
+<td>The game shall have characters, each with different statistics(F6) defining what they are (enemies, friendlies, etc).</td>
+</tr>
+<tr>
+<td>F6</td>
+<td>Statistics</td>
+<td>The gameshall have <b>statistics</b> for the player and other characters in the game. Statistics will be:
+<ul>
+<li>Health Points</li>
+<li>Debuffs (hangover, wounds, bleeding)</li>
+<li>Aggressiveness</li>
+</ul>
+</td>
+</tr>
+<tr>
+<td>F7</td>
+<td>Combat</td>
+<td>The game shall have a simple <b>combat system.</b> If the player is able to attack a target, depending on the aggressiveness of the target, it can
+attack the player back. If the attack hits there is 50/50 chance of damage. The combat system should have a modifier mechanic on which the chance
+of hitting or damage depends on. "Stronger" enemies have higher modifiers thus will give more damage to the player. </td>
```

Comment:
If the chance to do damage is predetermined to be 50/50, why do you need a modifier to calculate it? Looks like the modifier should only calculate the damage number. It's unclear if these rules apply only to the attacking enemies or the player as well.